types/src/front/assistant/agent.ts

```diff
@@ -140,6 +140,21 @@ export type AgentUsageType = {
   timePeriodSec: number;
 };
 
+export type LightAgentUsageType = {
```

Comment:
cf other [thread](https://github.com/dust-tt/dust/pull/5649#discussion_r1644209603) in this PR, i think the code would be simpler if we can avoid another Light/heavy type split

Reply:
@philipperolet following on [this](https://github.com/dust-tt/dust/pull/5649#discussion_r1644304251) would you rather make `usersWithAgentInListCount` and `userCount` fields optional?